deserialize/type_checks.py

```diff
@@ -61,6 +61,20 @@ def is_list(type_value):
         return False
 
 
+def is_tuple(type_value):
```

Comment:
Can you move this below `list_content_type`? That way everything sticks together.